proto/cosmos/group/v1beta1/types.proto

```diff
@@ -53,9 +52,23 @@ message PercentageDecisionPolicy {
   // percentage is the minimum percentage the weighted sum of yes votes must meet for a proposal to succeed.
   string percentage = 1;
 
-  // timeout is the duration from submission of a proposal to the end of voting period
-  // Within these times votes and exec messages can be submitted.
-  google.protobuf.Duration timeout = 2 [(gogoproto.stdduration) = true, (gogoproto.nullable) = false];
+  // windows defines the different windows for voting and execution.
+  DecisionPolicyWindows windows = 2;
+}
+
+// DecisionPolicyWindows defines the different windows for voting and execution.
+message DecisionPolicyWindows {
+  // voting_period is the duration from submission of a proposal to the end of voting period
+  // Within this times votes can be submitted with MsgVote.
+  google.protobuf.Duration voting_period = 1 [(gogoproto.stdduration) = true, (gogoproto.nullable) = false];
+
+  // min_execution_period is the minimum duration after the proposal submission
+  // where members can start sending MsgExec. This means that the window for
+  // sending a MsgExec transaction is:
+  // `[ submission + min_execution_period ; submission + voting_period + max_execution_period]`
```

Comment:
What if someone's set a decision policy such that min_execution_period > voting_period + max_execution_period? Should we check this somehow?

Reply:
I thought about this too, but didn't know where to put this check in the current PR.

I don't think this introduces any security or logic error though. It's just that all the proposals that this group account might create will be invalid, and will be caught late. So basically this group account sabotaging its own tx fees and time.

I can think of:
1. Updating the proto comments to reflect this.
2. In DecisionPolicy interface, change:
```diff
- Validate(g GroupInfo) error
- Validate(g GroupInfo, config Config) error // We do the check early, in Validate.
```

I think 1 is enough, but if you think 2 is worth it, i'll make the changes.

Reply:
Agreed, thanks!

Reply:
Done

Reply:
sorry if it wasn't clear, could you also update `Validate`?